docs/docs/getting-started/metrics.md

```diff
@@ -21,71 +21,202 @@ The Keptn Metrics Server unifies and standardizes access to all this data.
+
+     ```shell
+     prometheus_http_requests_total{code="200", container="prometheus", \
+          endpoint="http-web", handler="/-/healthy", \
+          job="kube-prom-stack-kube-prome-prometheus", namespace="default", \
+          service="kube-prom-stack-kube-prome-prometheus"}
+     ```
+
+     This is your metric and value as it is stored in Prometheus.
+
+## Configure Keptn for your metrics
+
+Now you must tell Keptn about the metrics you are using.j
+You do this by defining:
+
+- A
+  [KeptnMetricsProvider](../reference/crd-reference/metricsprovider.md)
+  resource to define the external observability platform you are using
+  as a data source.
+  For this exercise, this is the Prometheus server
+  you installed and exposed above.
+- A
+  [KeptnMetric](../reference/crd-reference/metric.md)
+  resource to define each metric query you want to pull.
+
+The steps are:
+
+1. Create a
+   [KeptnMetricsProvider](../reference/crd-reference/metricsprovider.md/)
+   resource for the observability platform you are using as a data source.
+   To do this, create a new `.yml` file with content like the following:
+
+      ```shell
+        apiVersion: metrics.keptn.sh/v1
+        kind: KeptnMetricsProvider
+        metadata:
+          name: local-prometheus
+          namespace: default
+        spec:
+          type: prometheus
+          targetServer: "http://prometheus-operated.default.svc.cluster.local:9090/"
+      ```
+
+     You can specify a virtually unlimited number of providers,
+     including multiple instances of each observability platform.
+     Each one must be assigned a unique `name`,
+     identified by the `type` of platform it is
+     and the URL of the target server.
+     If the target server is protected by a Secret,
+     provide information about the token and key.
+
+     Keptn uses the `name` you assign to reference this data source.
+
+     The `targetServer` field tells Keptn where to find this data source;
+     in this case, it points to the UI on port 9090.
+
+     Apply this file with the following command:
+
+     ```shell
+      kubectl apply -f YOUR-KEPTN-METRIC-PROVIDER.yml
+      ```
+
+1. Define your
+   [KeptnMetric](../reference/crd-reference/metric.md/)
+   custom resource for each piece of data you want to pull
+   (in this case, the Prometheus query for the metric you selected).
+
+     This is where you use the Prometheus query from before.
+     Note: The namespaces of `KeptnMetricsProvider` and `KeptnMetric` must match.
+
+     ```yaml
+     apiVersion: metrics.keptn.sh/v1
+     kind: KeptnMetric
+     metadata:
+       name: prometheus-http-requests-total
+       namespace: default
+     spec:
+       provider:
+         name: local-prometheus
+       query: 'prometheus_http_requests_total{code="200", handler="/-/healthy", job="kube-prom-stack-kube-prome-prometheus", namespace="default", service="kube-prom-stack-kube-prome-prometheus"}'
+       fetchIntervalSeconds: 10
+     ```
```

Comment:
let's move this to a separate file and use include

Reply:
See above.